espflash/src/image_format.rs

```diff
@@ -4,6 +4,7 @@ use std::{borrow::Cow, io::Write, iter::once, mem::size_of};
 
 use bytemuck::{bytes_of, from_bytes, Pod, Zeroable};
 use esp_idf_part::{Partition, PartitionTable, Type};
+use log;
```

Comment:
```suggestion
use log::debug;
```

To be consisten with other places of the code.